security/firely-auth/firely-auth-management-app.rst

```diff
@@ -3,122 +3,18 @@
 Firely Auth User Management
 ===========================
 
-When using the SQL Server user store, you can manage the users therein with a command line utility called the 'Firely Auth Management app'. This can be used both interactively from the commandline and automated from a script or devops pipeline.
+.. note:: 
+  In Firely Auth versions < 4.0.0 there was an executable to manage users, this has been deprecated.
 
-.. _firely_auth_mgmt_exe:
+After you have installed, started and created your admin account, you can now manage users in the UI.
+After creation of a non-sso account, an email will be sent to the address of the account to activate the account. When this link would be expired, you can resend this email through the UI as well.
 
-Executable
-----------
+When you want to manage the users programmatically, you can use the API's that are specified in the swagger documentation which you can find at: https://localhost:5001/swagger/ (or replace localhost with the url of your Firely Auth installation).
+To be able to use the api, you will need a client token (obtained trough the client_credentials flow) for a client who has the ``AllowManagementApiAccess`` property set to true.
+It is recommended to use a specific client for this API access.
```

Comment:
How about adding a default appsetting here which can be added via copy&paste?

Reply:
done